src/components/pages/RoleApply/Applications/Mentee.js

```diff
@@ -48,13 +48,16 @@ const initialFormValues = {
   validate_status: 'pending',
 };
 
-const Mentee = ({ dispatch, error, successPage }) => {
+const Mentee = ({ dispatch, error }) => {
   const { formValues, handleChange } = useForms(initialFormValues);
   const { push } = useHistory();
 
   const formSubmit = () => {
-    dispatch(postNewMenteeAccount(formValues));
-    push('/apply/success');
+    dispatch(postNewMenteeAccount(formValues))
+      .then(res => {
```

Comment:
great job adding the .then and .catch to this submit function!

Reply:
thank you